src/components/search-results.css

```diff
@@ -17,6 +17,11 @@
   text-decoration: none;
 }
 
+.search-results .search-results_active-tag {
```

Comment:
Nja, active i dette tilfellet er ikke en modifier. Det er en h1 på toppen av søkeresultatet som viser frem aktiv tag. Det finnes for eksempel ikke en search-results_tag